engine/table/src/main/java/io/deephaven/engine/table/impl/TableDefaults.java

```diff
@@ -223,6 +229,89 @@ default Table formatColumns(String... columnFormats) {
         return updateView(selectColumns);
     }
 
+    default void validateColor(String color) throws IllegalArgumentException {
```

Comment:
Changed this to be `private void validate...` instead so that it isn't exposed to API but still within TableDefaults. Not sure if the BaseTable approach is preferred in this scenario